src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -48,19 +50,22 @@
     private final ServiceConfigProvider serviceConfigProvider;
     private final TransformationClient transformationClient;
     private final AuthTokenGenerator s2sTokenGenerator;
-    private final CoreCaseDataApi ccdApi;
+    private final CoreCaseDataApi coreCaseDataApi;
+    private final CcdApi ccdApi;
 
     public CreateCaseCallbackService(
         CreateCaseValidator validator,
         ServiceConfigProvider serviceConfigProvider,
         TransformationClient transformationClient,
         AuthTokenGenerator s2sTokenGenerator,
-        CoreCaseDataApi ccdApi
+        CoreCaseDataApi coreCaseDataApi,
+        CcdApi ccdApi
```

Comment:
We so need a refactor on ccd api usage 😭. just a moan here

Reply:
what happened here? CCD == CoreCaseData

Reply:
`ccdApi` is in-house wrapper of `coreCaseDataApi` feign client. former has search functionality. feign was used directly here because no separate methods are available in the wrapper so it'll have yet another scope of start/submit event calls. As per my moan above ^^ - we desperately need refactor so usage is single point of entry

Reply:
or `getCaseRefsByBulkScanCaseReference` could have been added to `CoreCaseDataApi` class...

Reply:
That's feign client in library